snowpack/src/config.ts

```diff
@@ -21,7 +21,7 @@ import {
 import {addLeadingSlash, addTrailingSlash, NATIVE_REQUIRE, removeTrailingSlash} from './util';
 
 const CONFIG_NAME = 'snowpack';
-const ALWAYS_EXCLUDE = ['**/node_modules/**/*', '**/*.d.ts'];
+const ALWAYS_EXCLUDE = ['**/node_modules/**/*', '**/_*.{sass,scss}', '**/*.d.ts'];
```

Comment:
I feel pretty confident that any `.scss` or `.sass` file prefixed by an underscore can be ignored. This is a pretty strong convention within Sass—the whole intent is to exclude these from build.

Reply:
tentative +1 here, as long as we can keep this manageable. I don't want to get to a place where this is 10+ items long, including every convention from every ecosystem :)